g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/utils/S3Util.java

```diff
@@ -88,18 +87,16 @@ public Locker(String basePath, SingleComponentDTO compDTO) {
 		}
 
 		public boolean lockFile() {
-			long endTime = System.currentTimeMillis() + waitToLock;
 			String content = Double.toString(random.nextDouble());
-
+             int time=0;
 			do {
-				if (!waitLockfileDisappeared(endTime)) {
+				if (!waitLockfileDisappeared()) {
 					return false;
 				}
 
 				// Write the lock file
 				s3Client.putObject(this.key, content);
 				TimeUtils.sleep(waitS3Operation); // Wait for a while to let S3 finish writing.
```

Comment:
This line could be deleted because of strong consistency.

Reply:
done